### PR TITLE
Replace libsass with dart sass

### LINKED ISSUE #275: Remove dependency on ruby-sass which has reached end of life
ruby-sass has reached end of life, see:

- https://github.com/sass/ruby-sass
- https://sass-lang.com/blog/ruby-sass-is-unsupported

This project still depends on sass-rails 5.1.0 which depends on ruby-sass: https://github.com/sul-dlss/relevancy_dashboard/blob/master/Gemfile.lock#L312

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #275

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -76,17 +75,18 @@ group :deployment do
 end
 
 group :production do
-  gem 'sidekiq', '~> 5.0'
+  gem 'sidekiq'#, '~> 5.0'
 end
```
Comment (thatbudakguy) [inline Gemfile:79]: is this change related to this PR?

Comment (jcoyne) [reply Gemfile]: Nope, I've removed it.
